resources/views/inputs/input-error.blade.php

```diff
@@ -0,0 +1,9 @@
+@props ([
```

Comment:
if it requires moving the component, then isn't this the same as adding it as explicit component still? since you have now moved it to make use of it, while all the other includes are still in the same place 🤔

Reply:
It doesn't require moving the component directory-wise, I just moved it out of the `includes` as this component makes more sense to me to not be a part of that `includes` directory. I can just leave it in the `includes` directory, it's fine by me, only then we'd reference it like:

`<x-ark::inputs.includes.input-error />`

I'm fine by both, made more sense for me, for this component, to not be in includes.